spark/interpreter/src/main/java/org/apache/zeppelin/spark/Utils.java

```diff
@@ -178,4 +184,20 @@ public static String getUserName(AuthenticationInfo info) {
     }
     return uName;
   }
+
+  public static void printDeprecateMessage(SparkVersion sparkVersion,
+                                            InterpreterContext context,
+                                            Properties properties) throws InterpreterException {
+    context.out.clear();
+    if (sparkVersion.olderThan(SparkVersion.SPARK_2_2_0)
+            && Boolean.parseBoolean(
+                    properties.getProperty("zeppelin.spark.deprecatedMsg.show", "true"))) {
+      try {
+        context.out.write(DEPRRECATED_MESSAGE);
+        context.out.write("%text ");
+      } catch (IOException e) {
```

Comment:
kinda odd to get IOException here?

Reply:
Indeed, but I have to catch this message here.